LiteDB/Engine/SystemCollections/SysDatabase.cs

```diff
@@ -10,45 +10,44 @@ public partial class LiteEngine
     {
         private IEnumerable<BsonDocument> SysDatabase()
         {
-            var doc = new BsonDocument();
-
-            doc["name"] = _disk.GetName(FileOrigin.Data);
-            doc["encrypted"] = _settings.Password != null;
-            doc["readOnly"] = _settings.ReadOnly;
-
-            doc["lastPageID"] = (int)_header.LastPageID;
-            doc["freeEmptyPageID"] = (int)_header.FreeEmptyPageList;
-
-            doc["creationTime"] = _header.CreationTime;
-
-            doc["dataFileSize"] = (int)_disk.GetLength(FileOrigin.Data);
-            doc["logFileSize"] = (int)_disk.GetLength(FileOrigin.Log);
-            doc["asyncQueueLength"] = _disk.Queue.Length;
-
-            doc["currentReadVersion"] = _walIndex.CurrentReadVersion;
-            doc["lastTransactionID"] = _walIndex.LastTransactionID;
-
-            doc["cache"] = new BsonDocument
+            yield return new BsonDocument { ["_id"] = "name", ["value"] = _disk.GetName(FileOrigin.Data) };
+            yield return new BsonDocument { ["_id"] = "encrypted", ["value"] = _settings.Password != null };
+            yield return new BsonDocument { ["_id"] = "readOnly", ["value"] = _settings.ReadOnly };
+            yield return new BsonDocument { ["_id"] = "lastPageID", ["value"] = (int)_header.LastPageID };
+            yield return new BsonDocument { ["_id"] = "freeEmptyPageID", ["value"] = (int)_header.FreeEmptyPageList };
+            yield return new BsonDocument { ["_id"] = "creationTime", ["value"] = _header.CreationTime };
+            yield return new BsonDocument { ["_id"] = "dataFileSize", ["value"] = (int)_disk.GetLength(FileOrigin.Data) };
+            yield return new BsonDocument { ["_id"] = "logFileSize", ["value"] = (int)_disk.GetLength(FileOrigin.Log) };
+            yield return new BsonDocument { ["_id"] = "asyncQueueLength", ["value"] = _disk.Queue.Length };
+            yield return new BsonDocument { ["_id"] = "currentReadVersion", ["value"] = _walIndex.CurrentReadVersion };
+            yield return new BsonDocument { ["_id"] = "lastTransactionID", ["value"] = _walIndex.LastTransactionID };
+
+            yield return new BsonDocument
             {
-                ["extendSegments"] = _disk.Cache.ExtendSegments,
-                ["memoryUsage"] = 
-                    (_disk.Cache.ExtendSegments * MEMORY_SEGMENT_SIZE * PAGE_SIZE) +
-                    (40 * (_disk.Cache.ExtendSegments * MEMORY_SEGMENT_SIZE)),
-                ["freePages"] = _disk.Cache.FreePages,
-                ["readablePages"] = _disk.Cache.GetPages().Count,
-                ["writablePages"] = _disk.Cache.WritablePages,
-                ["pagesInUse"] = _disk.Cache.PagesInUse,
+                ["_id"] = "cache",
```

Comment:
Convert to 
"cache.extendSegments" 
"cache.memoryUsage"
"cache.freePages"
...

transactions too